.github/workflows/pythonapp.yml

```diff
@@ -108,16 +110,29 @@ jobs:
 
     - name: Testing setup
       run: |
+        # sudo mkdir -p /tmp/.onkodicom
+        # sudo chmod 777 /tmp/.onkodicom
         sudo apt-get update
-        sudo apt-get install xvfb libxkbcommon-x11-0 herbstluftwm libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 libopengl0 freeglut3 freeglut3-dev
+        sudo apt-get install xvfb libxkbcommon-x11-0 herbstluftwm libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 libopengl0 libglut3.12 libglut-dev freeglut3-dev libgeos-c1v5 libgeos++-dev libgeos-dev libxcb-cursor0
+        # install everything related to xcb.  libxcb-cursor0 is required for Qt >=6.5 and it has numerous dependencies
+        sudo apt-get install -y libxcb1 libxcb-cursor0 libxcb-glx0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render0 libxcb-render-util0 libxcb-shape0 libxcb-shm0 libxcb-sync1 libxcb-util1 libxcb-xfixes0 libxcb-xinerama0 libxcb-xkb1
```

Comment:
libxcb-cursor0 is required for Qt >=6.5.
There are other libxcb dependencies as well, and this was a kitchen sink approach to including all of them.